packages/core/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think we dont need a config for every package. could you solve it with one central tsconfig?

Reply:
Unfortunately, no, as `flatModuleOutFile` and `flatModuleId` both take a single value (https://angular.io/guide/aot-compiler#flatmoduleoutfile).